### DISCUSSION TITLE
Previous Beta versions

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#514] [Q&A] [answered]
Sometimes Beta users want to return to the previous version for a while. 
However a return to the previous version is then practically not possible.
With the Beta versions themselves, however, you can easily return anyway.
Maybe offer the first one in the line as an unmodified previous version ?

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi thanks for the suggestion.

Do you mean in Dropbox?

We could keep all Beta versions until their testing period ends.

**Reply by bios4** [reply]
I have all beta APKs saved (on my phone or in the cloud) until the "final" of a new version is available.
This should be doable for all (beta) users, so I really don't see the necessity to have all betas in your Dropbox share...

But if you are fine with it, go ahead.

**Reply by 0709wiwiwi** [reply]
Perhaps my question was redundant. 
It seems I can still install the version 3.042 apk.
I tried to downgrade once and it didn't work. Has something changed ?

**Reply by devemux86** [reply]
Current release remains compatible with Beta versions of the next release.
Android usually requires an uninstall first to install an older version.

Anyway, it is recommended not to install old versions over new ones. :slightly_smiling_face: 
Apps are tested to remain compatible as move forward (not backward).
(all platforms, all software)
